2/contrib/s2i/run

```diff
@@ -28,11 +28,25 @@ function create_jenkins_config_xml() {
     if [ -f "${image_config_path}.tpl" ]; then
       export KUBERNETES_CONFIG=$(generate_kubernetes_config)
       echo "Generating kubernetes-plugin configuration (${image_config_path}.tpl) ..."
+      export ADMIN_MON_CONFIG=$(generate_administrative_monitors_config)
+      echo "Generating administrative monitor configuration (${image_config_path}.tpl) ..."
       envsubst < "${image_config_path}.tpl" > "${image_config_path}"
     fi
   fi
 }
 
+function generate_administrative_monitors_config() {
+  DOLLAR='$'
+  if [[ "${DISABLE_ADMINISTRATIVE_MONITORS}" == "true" ]]; then
+      echo "
+        <string>hudson.model.UpdateCenter${DOLLAR}CoreUpdateMonitor</string>
```

Comment:
is this ${DOLLAR} right?  this value represents something that's being substituted into the config.xml.  I wouldn't expect it to need to be escaped.

Reply:
or rather, you can use echo with single quotes to avoid variable resolution here.

Reply:
Yes this works.

And so does `hudson.model.UpdateCenter\$CoreUpdateMonitor`, without setting of `DOLLAR='$'` on the same line as the `envsubst` call

I employed the current approach based on the discussion that occurred in the original PR, but am perfectly find just switching to `hudson.model.UpdateCenter\$CoreUpdateMonitor`

What does not work:  A string of `hudson.model.UpdateCenter$CoreUpdateMonitor`

Followed later on by `DOLLAR='$' envsubst....` ... the `$` is lost on the above echo even and then there is nothing for `DOLLAR='$' envsubst ...`

Reply:
> the $ is lost on the above echo 

single quotes would solve that.

```
#!/bin/sh
  
function foo() {
  echo 'a$b'
}
export bar=$(foo)
envsubst < file.json
```

```
$ cat file.json 
a
$bar
foo
```

```
$ ./t.sh 
a
a$b
foo
```